features/hangouts/edit_hangout_url.feature

```diff
@@ -30,4 +30,12 @@ Feature: Manual Edit of Hangout URL
     And I manually set a hangout link for event "Repeat Scrum"
     Then "Repeat Scrum" shows live for that hangout link for the event duration
     And "Repeat Scrum" is not live the following day
-    And there should be exactly 3 hangouts
\ No newline at end of file
+    And there should be exactly 3 hangouts
+
+  Scenario: Event doesn't go live before Hangout URL is updated
+    Given the date is "2014 Feb 5th 6:00am"
+    And I manually set a hangout link for event "Repeat Scrum"
+    Given the date is "2014 Feb 5th 7:05am"
```

Comment:
@tansaku I had a hard time writing the cucumber test because it was also passing with the fix in #1694. I then realized that bug only happens unless [`@recent_hangout`](https://github.com/AgileVentures/WebsiteOne/blob/1eaccb5782e02325bd6797962bc94037111f9deb/app/controllers/events_controller.rb#L13) is not nil. In real application it doesn't stay nil but in the test it was staying nil. I couldn't understand what exactly is required to make it not nil. So I ended up adding lines line 36 and 37 above and line 115 in features/step_definitions/hangout_steps.rb